cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -94,12 +98,18 @@ spec:
             - --disable-cert-rotation={{ .Values.controllerManager.disableCertRotation }}
             - --max-serving-threads={{ .Values.maxServingThreads }}
             - --tls-min-version={{ .Values.controllerManager.tlsMinVersion }}
+            - --client-ca-name={{ .Values.controllerManager.clientCAName }}
```

Comment:
All of these flags should be set conditionally only when the helm variable is set. We do not want to set new flags to user deployments during upgrades. Additionally, please add the helm variable information in https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/static/README.md as well to make it visible.